internal/controlplane/handlers_authz.go

```diff
@@ -111,7 +121,7 @@ func populateEntityContext(
 	authzClient authz.Client,
 	req any,
 ) (context.Context, error) {
-	projectID, err := getProjectIDFromContext(req)
+	projectID, err := getProjectIDFromRequest(req)
```

Comment:
This rename was probably too much -- I can roll it back if you want (`Context` being an overloaded name between the request field and the Golang concept).

Reply:
nah, it's fine